contracts/DAITokenWrapper.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity 0.7.1;
+
+
+import "./interfaces/ITokenWrapper.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
+
+/**
+ * @title DAITokenWrapper
+ * @notice Contract for wrapping call to DAI token permit function because the DAI token permit function has a different signature from other tokens with which the protocol integrates
+ */
+contract DAITokenWrapper is 
+    ITokenWrapper,
+    Pausable,
```

Comment:
Why is that contract Pausable?
The pause does not have any effect on it, and it doesn't look to be called at the protocol side... (expect if I've missed it)
Btw, even if the token contract is Pausable, we would not be able to pause it (not our own), so...

Reply:
I suppose it's not really needed. I'll remove Pausable.

Reply:
Removed